doc/sphinxman/source/introduction.rst

```diff
@@ -472,114 +472,85 @@ For more details, see Tables :ref:`Energy <table:energy_gen>`,
+    | CISD                    | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | QCISD                   | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | QCISD(T)                | RHF            | CONV              | ---          | ---            | threaded [#f3]_             |           | E          |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | CI(n)                   | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | FCI                     | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRPT2                | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD               | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Mk-MRCCSD(T)            | RHF/ROHF/TCSCF | CONV              | ---          | ---            | threaded [#f3]_             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | RAS-CI                  | RHF/ROHF       | CONV              | ---          | ---            | partially threaded          |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | CASSCF, RASSCF          | RHF/ROHF       | CONF/DF           | ---          | ---            | partially threaded          |           |            |
```

Comment:
@dgasmith, is CONF/DF right for casscf & rasscf? Any other changes amongst the detci-driven lines?